pkg/cloud/cloud.go

```diff
@@ -64,13 +64,17 @@ type FileSystem struct {
 	FileSystemId string
 	CapacityGiB  int64
 	DnsName      string
+	S3ImportPath string
```

Comment:
I checked controller `CreateVolume`, `S3ImportPath` and `S3ExportPath` is not needed. We don't need to add them in `FileSystem`

Reply:
Deleted.